docs/ghcr_packages.md

```diff
@@ -0,0 +1,30 @@
+# GHCR Packages
```

Comment:
Start with explanation of what GHCR is and give user a description how to use it.

Mention that our packages are private and can only be used from gitpod.io and workspaces. To my knowledge local usage is paid (which we want to avoid).

Reply:
Added explanation. Usage is in the other paragraph. 

As far as I know, local usage is free up to 1 GB of downloaded packages per month, but as you wish, I kept only gitpod.io and GitHub Codespaces in the docs.